src/components/manage/Widgets/__snapshots__/IdWidget.test.jsx.snap

```diff
@@ -181,7 +226,7 @@ exports[`IdWidget renders an id widget with invalid characters 1`] = `
           <div
             class="ui red pointing basic label"
           >
-            Only lowercase letters (a-z) without accents, numbers (0-9), and the two characters "-" and "_" are allowed.
+            Only lowercase letters (a-z) without accents, numbers (0-9), and the two characters "-" and "_", and dot "." are allowed.
```

Comment:
@reebalazs This is confusing now, it says two characters and then lists 3 characters.
```suggestion
            Only lowercase letters (a-z) without accents, numbers (0-9), and the characters "-", "_", and "." are allowed.
```

Reply:
Done.